recipes/imread/meta.yaml

```diff
@@ -0,0 +1,69 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "imread" %}
+{% set version = "0.6" %}
+{% set sha256 = "97364c44b9861ebd74dbf9a7cafea604a802543b3a9b463c9de68befed72d86c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - add_sys_prefix_include.patch
+
+build:
+  number: 0
+  # Patches for Windows compilation are welcome, currently fails to link
+  skip: true  # [win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    # When setuptools is available add the `--single-version-externally-managed --record record.txt` above.
+    - setuptools
+    # if your project compiles code (such as a C extension) then add `toolchain` as a build requirement.
+    - toolchain
+    - numpy x.x
+    - msinttypes  # [win]
+    - jpeg
+    - libpng
+    - libwebp
+    - libtiff
+  run:
+    - python
+    - numpy x.x
+    - jpeg
+    - libpng
+    - libwebp
+    - libtiff
+
+test:
+  imports:
+    - imread
+
+  requires:
```

Comment:
I don't think this `requires` section is necessary if you are just doing an import test, because you'll pull in all of the standard requirements through the `run` requirements above. If you were running a test suite that required additional testing dependencies, then you would include them here.